packages/foreman/foreman/foreman.spec

```diff
@@ -758,6 +763,7 @@ install -Dp -m0644 %{SOURCE4} %{buildroot}%{_sysconfdir}/cron.d/%{name}
 install -Dp -m0644 %{SOURCE5} %{buildroot}%{_tmpfilesdir}/%{name}.conf
 install -Dp -m0644 extras/systemd/%{name}.service %{buildroot}%{_unitdir}/%{name}.service
 install -Dp -m0644 extras/systemd/%{name}.socket %{buildroot}%{_unitdir}/%{name}.socket
+install -Dp -m0755 %{SOURCE6} %{buildroot}%{_datadir}/%{name}/bin/%{name}-sidekiq
```

Comment:
`/usr/share/foreman/bin` is an odd place for this. Granted, we already have some precedent.

Reply:
Yeah, the ideal place is `/var/lib/libexec/foreman` or something like that (for binaries which are not meant for end users). I can do this if you insist.

Reply:
I don't insist in this PR. If we do that, we should move everything from that bin dir to `/usr/libexec/foreman`. That probably takes a bit more coordinated effort.